docs/src/content/docs/ko/guides/components.mdx

```diff
@@ -229,6 +229,38 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="컴포넌트" href="/ko/guides/components/" />
 </CardGrid>
 
+### 링크 버튼
+
+시각적으로 구분되는 콜투액션 링크에는 `<LinkButton>` 컴포넌트를 사용합니다.
+링크 버튼은 가장 관련성이 높거나 실행 가능한 콘텐츠로 사용자를 안내하는 데 유용하며 랜딩 페이지에서 자주 사용됩니다
+
+`<LinkButton>`에는 [`href`](https://developer.mozilla.org/ko/docs/Web/HTML/Element/a#href) 속성이 필요하며, 'target'과 같은 다른 링크 속성을 선택적으로 허용합니다.
+
+`icon` 속성을 [Starlight 내장 아이콘 중 하나](#모든-아이콘)의 이름으로 설정하여 텍스트 옆에 아이콘을 포함할 수 있습니다.
+텍스트 앞에 아이콘을 배치하기 위해 `iconPlacement` 속성을 `start`로 설정할 수 있습니다 (기본값은 `end`).
+
+`primary` (기본값), `secondary`, `minimal`로 설정할 수 있는 `variant` 속성을 사용하여 링크 버튼의 모양을 사용자 정의합니다.
+
+```mdx
+# src/content/docs/example.mdx
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+<LinkButton href="/getting-started/">시작하기</LinkButton>
```

Comment:
```suggestion
<LinkButton href="/ko/getting-started/">시작하기</LinkButton>
```

We may as well use the `ko/` page so that if a user test it, they're still in the same language.

Reply:
Thank you!